webapp/channels/src/components/widgets/inputs/input/input.tsx

```diff
@@ -116,6 +133,28 @@ const Input = React.forwardRef((
     const handleOnChange = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
         setCustomInputLabel(null);
 
+        const newValue = event.target.value;
+
+        // Check for maxLength (limit) validation in real-time
+        if (limit && newValue && !Array.isArray(newValue) && newValue.toString().length > limit) {
```

Comment:
This case can't really happen because we pass the max length into the underlying textarea/input which means the browser would prevent the user from typing more characters than this.

I could see maybe having a message that's displayed when the string is the maximum length, but it can never be longer than that

Reply:
ok, no longer needed so was removed.